sources/editor/Stride.Editor/Stride.Editor.csproj

```diff
@@ -26,6 +26,7 @@
   </ItemGroup>
   <ItemGroup>
     <ProjectReference Include="..\..\assets\Stride.Core.Assets.CompilerApp\Stride.Core.Assets.CompilerApp.csproj" />
+    <ProjectReference Include="..\..\engine\Stride.Games\Stride.Games.csproj" />
```

Comment:
Kind of suspicious here as well, did you commit this by mistake ?

Reply:
right, not sure how that ref got added.  removing it and things still build